review-database/postgres/data-generation/users.generator.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const faker = require('faker');
+const AWS = require('aws-sdk');
+
+const generateUsersData = (n, start, amount) => {
+  const generateImage = () => `https://s3-us-west-1.amazonaws.com/review-sdc/img/profile${Math.floor(Math.random() * (1000)) + 1}.jpg`;
+
+  const write = fs.createWriteStream(`users${n}.data.csv`);
+  let id = start;
+  for (let i = start; i < start + amount; i += 1) {
+    let results = `${id}
```

Comment:
Again, might be clearer if you define each of these separately so we know what they represent. Not sure if it's worth implementing now though.